test/functional/feature_maxuploadtarget.py

```diff
@@ -30,11 +30,11 @@ def on_block(self, message):
         self.block_receive_map[message.block.sha256] += 1
 
 class MaxUploadTest(BitcoinTestFramework):
- 
+
     def set_test_params(self):
         self.setup_clean_chain = True
         self.num_nodes = 1
-        self.extra_args = [["-maxuploadtarget=800"]]
+        self.extra_args = [["-maxuploadtarget=800", "-blockmaxweight=3996000"]]
```

Comment:
May just remove these, since it's the default, but the limit was explicitly added in the original test.

Reply:
I agree that we should just remove this - there's no point in explicitly setting config to the default value (in fact it just distracts from the purpose of the test).